pkg/api/vlabs/types.go

```diff
@@ -519,6 +519,7 @@ type CustomCloudProfile struct {
 	AzureEnvironmentSpecConfig *AzureEnvironmentSpecConfig `json:"azureEnvironmentSpecConfig,omitempty"`
```

Comment:
This comment is out of place. It is based on you email. Given this input:

```
"customCloudProfile": {
  "identitySystem":"adfs",
  "authenticationMethod":"client_certificate",
  "connectionMode":"connected",
  "environment": {
    "name": "azureStackCloud",
    "resourceManagerEndpoint": "https://management.local.azurestack.external/"
  }
}
```
- Is it possible for `resourceManagerEndpoint` to deviate from this pattern: `https://management.{{location}}.azurestack.external/`?
- If not, then AKSe can compute it to further simplify the input cluster definition

Reply:
azurestack.external is input parameter during deploying azure stack.

Reply:
k, I see